lib/puppet/forge.rb

```diff
@@ -90,9 +90,9 @@ def search(term)
   # @see SemanticPuppet::Dependency::Source#fetch
   def fetch(input)
     name = input.tr('/', '-')
-    uri = "/v3/releases?module=#{name}"
+    uri = "/v3/releases?module=#{name}&sort_by=version"
     if Puppet[:module_groups]
-      uri += "&module_groups=#{Puppet[:module_groups].gsub('+', ' ')}"
+      uri += "&module_groups=#{Puppet[:module_groups]}"
```

Comment:
is there any downside to leaving these gsub's here in case something else is using this uri?

Reply:
Probably not?

Reply:
Re-added the `gsub`s.